tests/Unit/Helpers/ParallelAlgorithms/Interpolation/InterpolateOnElementTestHelpers.hpp

```diff
@@ -61,6 +64,22 @@ struct MultiplyByTwoCompute : MultiplyByTwo, db::ComputeTag {
 };
 }  // namespace Tags
 
+// compute_vars_to_interpolate for test.
+struct ComputeMultiplyByTwo {
+  template <typename SrcTagList, typename DestTagList>
+  static void apply(const gsl::not_null<Variables<DestTagList>*> target_vars,
+                    const Variables<SrcTagList>& src_vars,
+                    const Mesh<3>& /*mesh*/) {
+    static_assert(std::is_same_v<SrcTagList, tmpl::list<Tags::TestSolution>>,
+                  "SrcTagList must be only TestSolution");
+    static_assert(std::is_same_v<DestTagList, tmpl::list<Tags::MultiplyByTwo>>,
+                  "DestTagList must be only MultiplyByTwo");
```

Comment:
Can you make the arguments explicitly these types?

Reply:
Yes, done.